File: qualifier-week-3/non graded assignments/PPA2 - Not Graded.c
#include <stdio.h>
#include <math.h>
int main() {
    double principal, rate, time;
    scanf("%lf%lf%lf", &principal,&rate,&time);
    // Write solution code below
    double simpleInterest;
    simpleInterest = (principal * rate * time) / 100;
    printf("%.2lf", simpleInterest);
    return 0;
}

File: qualifier-week-3/non graded assignments/PPA2 - Not Graded.rs
fn main() {
    let mut principal: f64 = 0.0;
    let mut rate: f64 = 0.0;
    let mut time: f64 = 0.0;

    println!("Enter the principal amount: ");
    let _ = scanf!("{:f64}", &mut principal);

    println!("Enter the interest rate (in percentage): ");
    let _ = scanf!("{:f64}", &mut rate);

    println!("Enter the time period (in years): ");
    let _ = scanf!("{:f64}", &mut time);

    rate /= 100.0;

    let simple_interest = principal * rate * time;

    println!("Simple Interest: {:.2f}", simple_interest);
}
